alphabet/alphabet_contract.go

```diff
@@ -50,6 +51,10 @@ func _deploy(data interface{}, isUpdate bool) {
 		total          int
 	})
 
+	if args.notaryDisabled {
```

Comment:
You mean fail only if the original mode was notary, and if non-notary - just do nothing? I don’t like the unconditional transition to a notary mode, because update "request" can explicitly state the requirement "make it non-notary".

Reply:
This flag means nothing after the transition. `make is non-notary` is not a valid thing to request from 0.17.0+ contracts, we move to notary-only and that's it.

Reply:
> is not a valid thing

So we panic in this case, isn't that correct? I guess I missed your point.

Reply:
We can ignore this value (not use in any way) before we remove it completely (but this will take some time). Although panic may help us detect invalid arguments (`notaryDisabled == true`).